### ISSUE TITLE
The accuracy in Testing Dataset is higher than it in Training Dataset! WHY?

### ISSUE THREAD

**Issue by Antyaoyao** [kwotsin/TensorFlow-ENet#37]
The accuracy in Testing Dataset is higher than it in Training Dataset! WHY?
In my training dataset,I got accuracy at about 0.88,but in my testing dataset,I got it at about 0.9317,but why?
Thanks.
